tools/cmd/prepare_prebuilt_workers/main.go

```diff
@@ -105,41 +112,51 @@ func main() {
 	for _, pair := range languagesSelected {
 		split := strings.Split(pair, ":")
 
-		if len(split) != 2 || split[len(split)-1] == "" {
-			log.Fatalf("Input error in language and gitref selection, please follow the format as language:gitref, for example: c++:<commit-sha>")
+		if len(split) < 2 || split[len(split)-1] == "" {
+			log.Fatalf("Input error in language and gitref selection, please follow the format as language:repository:gitref, for example: c++:grpc/grpc/<commit-sha>")
```

Comment:
"selection. Please follow the format language:gitref or language:repository:gitref, for example c++:master or c++:grpc/grpc:master".